drivers/SmartThings/philips-hue/src/hue/api.lua

```diff
@@ -29,9 +29,20 @@ local ControlMessageBuilders = {
   end
 }
 
+local function try_send(instance, message)
+  if not instance._ctrl_tx then
+    log.error(st_utils.stringify_table(message, "Couldn't send the followings due to closed transmit channel", false))
+  end
+
+  local success, err = pcall(instance._ctrl_tx:send(message))
```

Comment:
I don't think this is the syntax you want.  I think this will try to call the function returned by the send command.

Reply:
Oof, yeah, bad copy/paste, good catch.